Years of experience and Salary dataset/Model/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Follow the README.md template and insert proper description of the project. Follow this for reference, https://github.com/abhisheks008/ML-Crate/blob/main/.github/readme_template.md